adoc/deployment-bare-metal.adoc

```diff
@@ -0,0 +1,73 @@
+include::entities.adoc[]
+:isofile: SLE-15-SP1-Installer-DVD-x86_64-Build225.1-Media1.iso
+:isolink: http://download.suse.de/install/SLE-15-SP1-Installer-GMC1/
+
+== Deployment on VMWare
```

Comment:
Is this related to vmware or baremetal?

Reply:
I created those for VMWare. If we have links for baremetal I would take them too. I want to be able to have links to the media that don't change much.

Reply:
i think it was resolved already by Klaven